Makefile

```diff
@@ -1,33 +1,30 @@
-ifdef TARGET
-TARGETDEFINED="true"
-else
-TARGETDEFINED="false"
-endif
-
 dev:
-	$(eval export env_stub=dev)
-	@true
+	echo "TODO: Remove dev function call from deploy-utils"
```

Comment:
Need to preserve this interface that deploy-utils depends on.
The variables it was setting "env_stub" is not required and we are deploying without them.

Will remove deploy utils from this soon.